src/htm/os/ImportFilesystem.hpp

```diff
@@ -54,8 +54,13 @@
   #endif
 #else
   // C++11
-  #include <boost/filesystem.hpp>
-  #define USE_BOOST_FILESYSTEM 1
+  #if (defined( __GNUC__ ) && __GNUC__ >= 8) || (defined(__clang_major__) && __clang_major__ >= 7)
+      #include <filesystem>
```

Comment:
not sure this is correct, imho users just should know to specify `-std=c++17` when compiling their app with htm.core. As you could have gcc-8.3 but use c++11/98 in your project and cannot use TS filesystem then.

Reply:
> imho users just should know to specify -std=c++17 when compiling their app with htm.core. 

Well, I did not know that.  I had to go figure it out when it kept trying to include boost.